presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -681,3 +681,44 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+
+Logging Properties
+------------------
+
+Presto uses the server application framework `Airlift
+<https://github.com/airlift/airlift>`_ and therefore supports its configuration
```

Comment:
This is an informative piece. At the same time, people may be tempted to click a link and expect any form of documentation there, which is not the case. 

@electrum ?

(i'm fine either way)

Reply:
Sure .. at the same time there is a LOT of information there if they can read the source ... so it should be mentioned somewhere and not hidden.

Reply:
@electrum ?

Reply:
i checked with @electrum and he stated Airlift should be considered implementation detail, not to be mentioned here.

Reply:
I disagree but will remove